src/controllers/vendorController.js

```diff
@@ -136,6 +147,11 @@ class VendorController {
         { active: false },
         { where: { id: existingVendor.dataValues.UserId } }
       );
+      const emailData={
+        email:existingUser.email,
+        report:reports[1].category
+      };
+      await sendEmail(emailData,"disableVendorAccount");
```

Comment:
- This await won't have any effect since the sendMail function is not declared as async function.

Reply:
let me remove it then